### PR TITLE
refactor unexpected changesets command (DAT-6637)

### PR DESCRIPTION
**Pull request by StevenMassaro**
## Impact

- [ ] Bug fix (non-breaking change which fixes expected existing functionality)
- [X] Enhancement/New feature (adds functionality without impacting existing logic)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: liquibase-standard/src/main/java/liquibase/Liquibase.java [hunk 1]
```diff
@@ -1178,70 +1178,26 @@ public void reportStatus(boolean verbose, Contexts contexts, LabelExpression lab
...
-                        out.append("     ").append(ranChangeSet.toString()).append(StreamUtil.getLineSeparator());
-                    }
-                }
-            }
-
-            out.flush();
-        } catch (IOException e) {
-            throw new LiquibaseException(e);
-        }
+        // todo need to fill out command here
```
Comment (StevenMassaro) [inline liquibase-standard/src/main/java/liquibase/Liquibase.java:1200]: I need to do this part! Oops

File: liquibase-standard/src/main/java/liquibase/Liquibase.java [hunk 2]
```diff
@@ -1178,71 +1178,38 @@ public void reportStatus(boolean verbose, Contexts contexts, LabelExpression lab
...
     public void reportUnexpectedChangeSets(boolean verbose, String contexts, Writer out) throws LiquibaseException {
         reportUnexpectedChangeSets(verbose, new Contexts(contexts), new LabelExpression(), out);
     }
 
+    @Deprecated
     public void reportUnexpectedChangeSets(boolean verbose, Contexts contexts, LabelExpression labelExpression,
                                            Writer out) throws LiquibaseException {
         changeLogParameters.setContexts(contexts);
         changeLogParameters.setLabels(labelExpression);
-        checkLiquibaseTables(false, getDatabaseChangeLog(true), null, null);
```
Comment (filipelautert) [inline liquibase-standard/src/main/java/liquibase/Liquibase.java:1217]: Seems we lost checkLiquibaseTables when moving the code to the command. In fact a lot of steps use it, couldn't we create a helper for it?

Comment (StevenMassaro) [reply liquibase-standard/src/main/java/liquibase/Liquibase.java:1217]: The command calls that method, here: https://github.com/liquibase/liquibase/blob/master/liquibase-standard/src/main/java/liquibase/command/core/helpers/DatabaseChangelogCommandStep.java#L85

**Review by filipelautert — APPROVED** [review]

File: liquibase-standard/src/main/java/liquibase/Liquibase.java (hunk 2 above)
Comment (filipelautert) [reply liquibase-standard/src/main/java/liquibase/Liquibase.java:1217]: Thanks, I missed that.

**Pull request merged**